data/json/items/book/misc_newage.json

```
@@ -0,0 +1,74 @@
+[
+  {
+    "id": "book_newage_generic",
+    "type": "BOOK",
+    "name": { "str": "new-age book", "str_pl": "new-age books" },
```

```suggestion
    "name": { "str": "new-age book" },
```
Plural defaults to singular form + "s".